src/modules/transaction/mutations/TransactionCreateMutation.ts

```diff
@@ -27,17 +28,23 @@ const mutation = mutationWithClientMutationId({
 
     const transaction = await new TransactionModel({transactionId, name, category, price}).save();
 
+
     if (!transaction) {
       return {
         error: 'Transaction not found',
       };
     }
 
+    await pubSub.publish("NEW_TRANSACTION", { transactionId: transaction._id})
```

Comment:
move `NEW_TRANSACTION` to a const

`const SUBSCRIPTIONS = { NEW_TRANSACTION: 'NEW_TRANSACTION'}`